test/PerfHint/arguments1.baseline

```diff
@@ -1,7 +1,7 @@
-PerfHint: Not optimized : Arguments object not optimized due to formals {
+PerfHint: Not optimized : Arguments object not optimized {

```

Comment:
I tried adding a PerfHint parser, but i was however not successful in figuring out the bytecode offset (probably this is not computed when I am accessing it!)